smdebug/core/hook.py

```diff
@@ -734,7 +734,7 @@ def _write_histogram_summary(self, tensor_name, tensor_value, save_collections):
                     continue
                 elif s_col.save_histogram is True:
                     np_value = self._make_numpy_array(tensor_value)
-                    if self.dry_run or np_value.dtype == np.bool or np_value.nbytes == 0:
+                    if self.dry_run or np_value.dtype == np.bool_ or np_value.nbytes == 0:
```

Comment:
Is there a reason we can't use the built in types instead? That seems to be recommended by numpy https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated

Reply:
actually, I guess we don't have good enough knowledge of test coverage to know whether we can use built-in types or not